models/pipeline.js

```diff
@@ -62,7 +67,7 @@ module.exports = {
      * @type {Joi}
      */
     get: Joi.object(mutate(MODEL, [
-        'id', 'scmUri', 'createTime', 'admins'
+        'id', 'scmUri', 'scmContext', 'createTime', 'admins'
```

Comment:
Does this mean it will break once we pull in this new data-schema? I'm wondering if we should put this as `optional` field first.